src/test/java/de/bonndan/nivio/input/DirectoryWatcherTest.java

```diff
@@ -39,7 +39,7 @@ public void testFileChange() throws IOException, InterruptedException, TimeoutEx
 
         Thread.sleep(1000);
         Files.write(tempFile.toPath(), "Hallo".getBytes(), StandardOpenOption.CREATE, StandardOpenOption.APPEND, StandardOpenOption.WRITE);
-        Thread.sleep(1000);
+        Thread.sleep(50000);
```

Comment:
Can't we use SystemUtils.IS_OS_MAC_OSX here? Or a lower value? 50 seconds build delay is a bit much for regular CI builds.

Reply:
Sure. Although I am not sure yet why we are seeing this behaviour. Of course it comes to mind, that it is related to the different file systems. I would still want to check out the build without the Thread.sleep() on e.g. Windows 10 and Windows Server 2016.

You are running the build on your machine on what system? Linux?

In any case, we should investigate the underlying issue, if it is not already fixed with the mentioned PR.